go/app/main.go

```diff
@@ -151,19 +155,70 @@ func getItem(c echo.Context) error {
 	if err != nil {
 		errMessage(c, err, http.StatusBadRequest, "Unable to conveert item_id to int")
 	}
-	itemsData, err := readFile(c, "items.json")
+	db, err := sql.Open("sqlite3", "../db/mercari.sqlite3")
```

Comment:
Optional: because this code repeats, you can use a `const` or `func` to replace it.